flake8_spellcheck/__init__.py

```diff
@@ -166,6 +166,8 @@ def _parse_token(self, token_info):
         elif (
             token_info.type == tokenize.COMMENT
             and "comments" in self.spellcheck_targets
+            # ignore flake8 pragma comments
+            and token_info.string.lstrip("#").split()[0] != "noqa:"
```

Comment:
I'm guessing the reason why this is not just `startswith("# noqa")` is to cater for varying whitespace length? 

Could you add a test for that too if its the case since I would normally assume `startswith` is the simplest approach

Reply:
I'll be honest, I didn't even know `startswith` was a method ^^"

but yes, imo if the pragma is valid to flake8, it should be valid to the spellchecker. just my opinion though